Ghidra/Processors/Sparc/data/languages/SparcV9.sinc

```diff
@@ -1070,7 +1131,11 @@ TICC:  "%xcc"  is cc1_4=1 &cc0_4=0 { }
 trap: RS1+RS2     is i=0 & RS1 & RS2    { local tmp = ((RS1 + RS2)    & 0x7F); export tmp; }
 trap: RS1+swtrap  is i=1 & RS1 & swtrap { local tmp = ((RS1 + swtrap) & 0x7F); export tmp; }
 
-:t^tcc TICC, trap                    is op=0x2 & op3=0x3a & tcc & TICC & trap { sw_trap(trap); }
+:t^tcc TICC, trap                    is op=0x2 & op3=0x3a & tcc & TICC & trap
```

Comment:
Adding a slight modification to the trap so that it can be over-ridden not to return for a trap.
```
:t^tcc TICC, trap                    is op=0x2 & op3=0x3a & tcc & TICC & trap
{
	if (!tcc) goto inst_next;
	local target:$(SIZE) = sw_trap(trap);
	call [target];
}
```